content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+Type:                     NodePort
+IP Families:              <none>
+IP:                       10.97.222.217
+IPs:                      10.97.222.217
+Port:                     <unset>  27017/TCP
+TargetPort:               27017/TCP
+NodePort:                 <unset>  30946/TCP
+Endpoints:                10.192.3.58:27017
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:
+  Type    Reason  Age   From         Message
+  ----    ------  ----  ----         -------
+  Normal  HpeCp   25m   hpecp-agent  Created HPECP K8S service
+
+$ kubectl describe svc mongo-1
+Name:                     mongo-1
+Namespace:                mongo
+Labels:                   app=mongo
+                          controller-revision-hash=mongo-7648bd99c8
+                          hpecp.hpe.com/hpecp-internal-gateway=true
+                          statefulset.kubernetes.io/pod-name=mongo-1
+Annotations:              hpecp-internal-gateway/27017: epicgw.customer.hpe.net:10031
+Selector:                 app=mongo,controller-revision-hash=mongo-7648bd99c8,statefulset.kubernetes.io/pod-name=mongo-1
+Type:                     NodePort
+IP Families:              <none>
+IP:                       10.98.166.198
+IPs:                      10.98.166.198
+Port:                     <unset>  27017/TCP
+TargetPort:               27017/TCP
+NodePort:                 <unset>  32648/TCP
+Endpoints:                10.192.3.59:27017
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:
+  Type    Reason  Age   From         Message
+  ----    ------  ----  ----         -------
+  Normal  HpeCp   22m   hpecp-agent  Created HPECP K8S service
+
+$ kubectl describe svc mongo-2
+Name:                     mongo-2
+Namespace:                mongo
+Labels:                   app=mongo
+                          controller-revision-hash=mongo-7648bd99c8
+                          hpecp.hpe.com/hpecp-internal-gateway=true
+                          statefulset.kubernetes.io/pod-name=mongo-2
+Annotations:              hpecp-internal-gateway/27017: epicgw.customer.hpe.net:10035
+Selector:                 app=mongo,controller-revision-hash=mongo-7648bd99c8,statefulset.kubernetes.io/pod-name=mongo-2
+Type:                     NodePort
+IP Families:              <none>
+IP:                       10.101.219.63
+IPs:                      10.101.219.63
+Port:                     <unset>  27017/TCP
+TargetPort:               27017/TCP
+NodePort:                 <unset>  31216/TCP
+Endpoints:                10.192.4.208:27017
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:
+  Type    Reason  Age   From         Message
+  ----    ------  ----  ----         -------
+  Normal  HpeCp   23m   hpecp-agent  Created HPECP K8S service
+
+
+
+ 
+Execute Mongo Shell into any one of the mongo services to configure Mongo Replica-Set
```

Comment:
Is this another Step-? As it shows in the preview, the text seems quite different, almost as if it is another step. It may also be from a screenshot? I noticed that the s in "services" was highlighted in yellow in the preview, so I'm not sure. If it is a step, don't forget a short explanation just below before you get into all that code.

Reply:
Have addressed / updated based on previous commets.